x.toml

```diff
@@ -142,6 +142,9 @@ lazy_static = "use once_cell::sync::Lazy instead"
 criterion = "criterion is only for benchmarks"
 proptest = "proptest is only for testing and fuzzing"
 
+[workspace.direct-dep-dups]
```

Comment:
Can we leave a note here to make it clear to the committer and reviewer that the dupes are temporary and they should be clean up when the associated work is complete?